client/src/core/client/admin/App/Navigation/NavigationContainer.tsx

```diff
@@ -17,21 +18,22 @@ interface Props {
   viewer: ViewerData | null;
 }
 
-class NavigationContainer extends React.Component<Props> {
-  public render() {
-    return (
-      <Navigation
-        showDashboard={
-          !!this.props.viewer && can(this.props.viewer, Ability.VIEW_STATISTICS)
-        }
-        showConfigure={
-          !!this.props.viewer &&
-          can(this.props.viewer, Ability.CHANGE_CONFIGURATION)
-        }
-      />
-    );
-  }
-}
+const NavigationContainer: FunctionComponent<Props> = ({ viewer }) => {
+  const [{ dsaFeaturesEnabled }] = useLocal<NavigationContainerLocal>(
+    graphql`
+      fragment NavigationContainerLocal on Local {
+        dsaFeaturesEnabled
+      }
+    `
+  );
+  return (
+    <Navigation
+      showDashboard={!!viewer && can(viewer, Ability.VIEW_STATISTICS)}
+      showConfigure={!!viewer && can(viewer, Ability.CHANGE_CONFIGURATION)}
+      showReports={!!dsaFeaturesEnabled}
```

Comment:
following previous comment, I wonder if this needs to be given a similar limitation as the above `Ability` checks in `can` have for things like dashboard and config... probably a question for design. For now, the DSA flag is good enough to get this merged in!

Reply:
Yeah, once we know how we want to limit, exactly!